app/models/katello/concerns/hostgroup_extensions.rb

```diff
@@ -76,5 +76,5 @@ def inherited_ancestry_attribute(attribute)
 end
 
 class ::Hostgroup::Jail < Safemode::Jail
-  allow :content_source, :rhsm_organization_label
+  allow :content_source, :rhsm_organization_label, :subscription_manager_configuration_url
```

Comment:
If I'm not wrong, this is the key https://github.com/theforeman/foreman/blob/develop/lib/foreman/renderer.rb#L15

`safe_mode` creates a Box using self, which in the case of Host is fine, but on Hostgroup is not as you can't call that method on it. The scope when the template is rendered is `self`, just as when you write a method within an object.